justfile

```
@@ -127,3 +127,7 @@ babbage-integration-tests:
 # run conway integration tests via nix
 conway-integration-tests:
   just conway-integration-tests-match ""
+
+latency-bench:
+   cabal run -O2 -v0 cardano-wallet-benchmarks:latency -- \
+   --cluster-configs lib/local-cluster/test/data/cluster-configs
```

```suggestion
   --cluster-configs lib/local-cluster/test/data/cluster-configs

```

(Newline at end of file.)